packages/templates/src/raw/fitness-apps-ranked/template.json

```diff
@@ -1082,7 +1082,7 @@
           "lineHeight": 1.3,
           "textAlign": "initial",
           "padding": { "locked": true, "hasHiddenPadding": false },
-          "content": "<span style=\"font-weight: 700; color: #f1ff51\">For someone who’s just getting into a daily workout Daily Workouts Fitness is a solid-and-free -choice.</span>",
+          "content": "<span style=\"font-weight: 700; color: #f1ff51\">For someone who’s just getting into a daily workout FitFrame is a solid-and-free-choice.</span>",
```

Comment:
Add a comma after `daily workout` and remove the hyphen before `choice`

**Final sentence** - For someone who’s just getting into a daily workout, FitFrame is a solid-and-free choice.

Reply:
updated.